### ISSUE TITLE
TP-Link Omada-Controller Rock-On 

### ISSUE THREAD

**Issue by janholtzhausen** [rockstor/rockon-registry#332]
Modified the unify rock-on to run mbentley/omada-controller

**Comment by phillxnet** [comment]
@janholtzhausen Hello there.

So could you give some more context / explanation here. I'm assuming you are about to present a pull request?

**Comment by janholtzhausen** [comment]
Hi, thats right, still figuring out the correct procedure :)

**Comment by janholtzhausen** [comment]
I hope it's not too much of a mess..

**Comment by phillxnet** [comment]
@janholtzhausen 
Re:

> I hope it's not too much of a mess..

I wouldn't worry about that. ;)

We do have the following doc entry that may help with procedure:
https://rockstor.com/docs/contribute/contribute_rockons.html

**Comment by janholtzhausen** [comment]
There, I think I checked all the boxes.
The next one will be a cleaner commit, I promise.

**Comment by Hooverdan96** [comment]
@janholtzhausen I can see the Rockon in your forked repo itself, however you would still need to initiate the Pull Request from that branch so it becomes visible over here.

As you do this, quick comment on the description, I think at this point we're stopping to support the armv7 version, as that's still 32-bit and focus on amd64 and arm64 only. So you can take that out before creating the Pull Request.

**Comment by janholtzhausen** [comment]
Is there anything else I need to do?

**Comment by Hooverdan96** [comment]
@janholtzhausen, I don't think so, someone just needs to review it. i will see whether I can do it in the next few days.

**Comment by janholtzhausen** [comment]
There's a couple more services I can contribute, but I'm not sure there's any interest?

**Comment by Hooverdan96** [comment]
@janholtzhausen I would imagine there will be some interest, if you want to list the areas you're thinking about. I have tried to test your Rockon, but apparently I am running into some issues. I've successfully installed it outside of Rockstor, but I seem to be having some issues with my testbed for Rockons that I haven't figured out yet (ran into issues when testing other rockon submissions). Once I can resolve those, I will retest again. The lack of progress on these has more to do with available capacity than lack of interest. So, again thanks for your submission on the TP controller, we'll get it in.
